packages/core/types/ts3.6/effects.d.ts

```diff
@@ -459,10 +459,10 @@ export interface ChannelPutEffectDescriptor<T> {
  *   Promise as result, or any other value.
  * @param args An array of values to be passed as arguments to `fn`
  */
-export function call<Fn extends (...args: any[]) => any>(
-  fn: Fn,
-  ...args: Parameters<Fn>,
-): CallEffect<SagaReturnType<Fn>>
+export function call<Args extends any[], Return>(
```

Comment:
Twitter thread that explains this change: https://twitter.com/TitianCernicova/status/1501237698991898624